### PR TITLE
Fix style, Use Same Test Project

### PR DESCRIPTION
**Pull request by waprin**
Also fix error in README and change test project to this repo's project.

### CODE REVIEW

**Comment by tswast** [comment]
Thanks!

Checkstyle is failing for import orders, but I think that's because I need to pull in the latest version of java-repo-tools.

File: monitoring/v3/src/main/java/CreateCustomMetric.java [hunk 1]
```diff
@@ -40,12 +39,12 @@
...
 
 
-
-
 /**
- * Class to demonstrate creating a custom metric with Cloud Monitoring. This class provides a few
- * functions that create a custom GAUGE metric, writes a timeseries value to it, then reads that
- * metric's value back within the last 5 minutes to see the value written.
+ * Class to demonstrate creating a custom metric with Cloud Monitoring.
+ * <p/>
```
Comment (tswast) [inline monitoring/v3/src/main/java/CreateCustomMetric.java]: I think the end p tags `</p>` are optional in HTML5. I don't think I've seen many javadocs that use them.

File: monitoring/v3/src/test/java/ListResourcesTest.java [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
-
 import com.google.api.services.monitoring.v3.Monitoring;
-
 import org.junit.Before;
 import org.junit.Test;
 
 import java.io.ByteArrayOutputStream;
 import java.io.PrintStream;
 
+import static com.google.common.truth.Truth.assertThat;
```
Comment (tswast) [inline monitoring/v3/src/test/java/ListResourcesTest.java]: style guide says static imports should appear first

**Comment by tswast** [comment]
You can check yourself if it's passing the Maven checkstyle plugin by running `mvn verify`.

**Comment by waprin** [comment]
Yes sorry fixing checkstyle locally now

File: monitoring/v3/src/test/java/ListResourcesTest.java (hunk 2 above)
Comment (waprin) [inline monitoring/v3/src/test/java/ListResourcesTest.java]: done

File: monitoring/v3/src/main/java/CreateCustomMetric.java (hunk 1 above)
Comment (waprin) [inline monitoring/v3/src/main/java/CreateCustomMetric.java]: done

**Comment by waprin** [comment]
Checkstyle just passed, just uploaded new commit with p tags fixed.

**Comment by waprin** [comment]
Fixes #118

**Comment by waprin** [comment]
@tswast ready for review

**Comment by tswast** [comment]
Looks Good To Me.

Thank you, a big improvement. I'm happy to have the tests running in Travis now, too.

**Pull request merged**